scripts/exregional_process_lightning.sh

```diff
@@ -0,0 +1,207 @@
+#
+# link or copy background and grid files
+#
+#-----------------------------------------------------------------------
+
+cp_vrfy ${fixgriddir}/fv3_grid_spec          fv3sar_grid_spec.nc
+
+#-----------------------------------------------------------------------
+#
+# Link to the NLDN data
+#
+#-----------------------------------------------------------------------
+run_lightning=false
+filenum=0
+LIGHTNING_FILE=${LIGHTNING_ROOT}/vaisala/netcdf
+for n in 00 05 ; do
+  filename=${LIGHTNING_FILE}/${YYJJJHH}${n}0005r
+  if [ -r ${filename} ]; then
+  ((filenum += 1 ))
+    ln -sf ${filename} ./NLDN_lightning_${filenum}
+  else
+   echo " ${filename} does not exist"
+  fi
+done
+for n in 55 50 45 40 35 ; do
+  filename=${LIGHTNING_FILE}/${PREYYJJJHH}${n}0005r
+  if [ -r ${filename} ]; then
+  ((filenum += 1 ))
+    ln -sf ${filename} ./NLDN_lightning_${filenum}
+    run_lightning=true
+  else
+   echo " ${filename} does not exist"
+  fi
+done
+
+echo "found GLD360 files: ${filenum}"
+
+#-----------------------------------------------------------------------
+#
+# copy bufr table from fix directory
+#
+#-----------------------------------------------------------------------
+BUFR_TABLE=${FIXgsi}/prepobs_prep_RAP.bufrtable
+
+cp_vrfy $BUFR_TABLE prepobs_prep.bufrtable
+
+#-----------------------------------------------------------------------
+#
+# Build namelist and run executable
+#
+#   analysis_time : process obs used for this analysis date (YYYYMMDDHH)
+#   NLDN_filenum  : number of NLDN lighting observation files 
+#   IfAlaska      : logic to decide if to process Alaska lightning obs
+#   bkversion     : grid type (background will be used in the analysis)
+#                   = 0 for ARW  (default)
+#                   = 1 for FV3LAM
+#-----------------------------------------------------------------------
+
+cat << EOF > namelist.lightning
+ &setup
+  analysis_time = ${YYYYMMDDHH},
+  NLDN_filenum  = ${filenum},
+  grid_type = "${PREDEF_GRID_NAME}",
+  obs_type = "nldn_nc"
+ /
+EOF
+
+#
+#-----------------------------------------------------------------------
+#
+# Copy the executable to the run directory.
+#
+#-----------------------------------------------------------------------
+#
+exec_fn="process_Lightning.exe"
+exec_fp="$EXECdir/${exec_fn}"
+
+if [ ! -f "${exec_fp}" ]; then
+  print_err_msg_exit "\
+The executable specified in exec_fp does not exist:
+  exec_fp = \"${exec_fp}\"
+Build lightning process and rerun."
+fi
+#
+#
+#-----------------------------------------------------------------------
+#
+# Run the process
+#
+#-----------------------------------------------------------------------
+#
+
+if [[ "$run_lightning" == true ]]; then
+    PREP_STEP
+    eval $RUN_CMD_UTILS --account=${ACCOUNT} --nodes=1-1 --tasks-per-node=1 --time 00:30:00 ${exec_fp} ${REDIRECT_OUT_ERR} || print_err_msg "\  
+        Call to executable to run lightning (nc) process returned with nonzero exit code."
```

Comment:
This seems like more args than we typically pass a `$RUN_CMD variable`.

Reply:
I agree and I removed the args we don't need.

Reply:
I now realized those extra args were there for me to run this task without logging onto a compute node.